google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -21,17 +21,18 @@ plugin.compatibility.error.message=You are running potentially incompatible vers
 plugin.compatibility.error.name.and.version={0} ({1})
 plugin.compatibility.error.update.link=Please {0}click here{1} to ensure that you have updated all of your plugins.
 
-appengine.flex.tools.menu.item.label=Deploy to App Engine flexible environment...
+appengine.name=App Engine (BETA)
+appengine.tools.menu.item.label=Deploy to App Engine...
```

Comment:
Here's the menu items showing both:

![image](https://cloud.githubusercontent.com/assets/1735744/15556694/3fe9a5ca-229d-11e6-9c16-ffc9ebe5d86f.png)

Theirs opens the old plugin.